dataland-community-manager/src/main/kotlin/org/dataland/datalandcommunitymanager/api/RequestApi.kt

```diff
@@ -187,8 +187,19 @@ interface RequestApi {
     ): ResponseEntity<StoredDataRequest>
 
     /** A method for searching data requests based on filters.
+     * @param dataType If set, only the requests with are data type in dataType are returned
```

Comment:
.. with are data type .. seems like a typo

Reply:
Will be fixed.

Reply:
ok